paddle/fluid/platform/gpu_info.cc

```diff
@@ -19,7 +19,10 @@ limitations under the License. */
 #include "gflags/gflags.h"
 #include "paddle/fluid/platform/enforce.h"
 
-DEFINE_double(fraction_of_gpu_memory_to_use, 0.92,
+// fraction_of_gpu_memory_to_use cannot be too high on windows,
+// since the win32 graphic sub-system can occupy some GPU memory
+// which may lead to insufficient memory left for paddle
+DEFINE_double(fraction_of_gpu_memory_to_use, 0.5,
```

Comment:
Please reset this fraction in windows, Either one windows specific function or define macro can do it. 
Given that most of our user works in linux, 0.92 will give their best effort of speed, 0.5 will slow down their models.